config/vufind/config.ini

```diff
@@ -98,6 +98,25 @@ asset_pipeline_max_css_import_size = 0
 ; alternate_themes above.  When commented out, no drop-down theme list will display.
 ;selectable_themes = "standard:Standard Theme,mobile:Mobile Theme"
 
+; This array maps each theme to a list of color schemes that it supports.
+; Even if no selectable_color_schemes (below) are enabled for selection in VuFind
+; itself, the user's system (browser or OS) may be set to use dark mode globally.
+; VuFind will only respect a color scheme choice (whether from VuFind or the system)
+; if it's supported by the current theme. If not, the first color theme specified
+; here will be used, or 'light' if there is none specified.
+; If your local theme supports more than light mode, add it to the array below.
+supported_color_schemes[bootstrap5] = "normal,dark,light"
+supported_color_schemes[sandal5] = "light"
```

Comment:
Any thoughts on whether it's worth building dark-mode compatibility for sandal5?  All I can say is it seems it will be a lot more work than bootstrap5 will be, there are color specified all around that theme's scss files.